src/signals/settings/users/containers/Overview/hooks/useFetchUsers.js

```diff
@@ -24,16 +24,33 @@ const useFetchUsers = ({ page, pageSize } = {}) => {
     (async function fetchData() {
       setLoading(true);
 
+      const pageParams = [
+        page && `page=${page}`,
+        pageSize && `page_size=${pageSize}`,
+      ];
+
+      const filterParams = Object.keys(filters).reduce((acc, filter) => {
+        const value = filters[filter];
+
+        if (value) {
+          return [
+            ...acc,
+            `${filter}=${value}`,
+          ];
+        }
+
+        return acc;
+      }, []);
+
+      const queryParams = [
+        ...pageParams,
+        ...filterParams,
+      ]
+        .filter(Boolean)
+        .join('&');
+
       try {
-        const params = [
-          page && `page=${page}`,
-          pageSize && `page_size=${pageSize}`,
-        ]
-          .filter(Boolean)
-          .join('&');
-        const url = [configuration.USERS_ENDPOINT, params]
-          .filter(Boolean)
-          .join('/?');
+        const url = [configuration.USERS_ENDPOINT, queryParams].filter(Boolean).join('/?');
```

Comment:
`filter(Boolean` Shouldn't be necessary here any more, right? You're already filtering on line 49